src/it/unimi/dsi/fastutil/Function.java

```diff
@@ -40,22 +39,36 @@
  * @see java.util.Map
  */
 
-public interface Function<K,V> {
+public interface Function<K,V> extends java.util.function.Function<K,V> {
+
```

Comment:
Yes, but this method is inherited from another interface. It is not defined in this interface. In this interface we simply define the default version. So we must {@inheritDoc} the specification, and add the documentation about the implementation.